src/random_number.py

```diff
@@ -0,0 +1,8 @@
+"""
+  Write a program that generates a random number.
+  Output:
+  The random number is: 4
+"""
+import random
+
+print("The random number is:", random.randint(0, 10))
```

Comment:
potresti estrarre la parte di codice che genera il numero random in una funzione che prende in input l'intervallo di valori 1,10